build.gradle

```diff
@@ -19,6 +19,15 @@ buildscript {
 plugins {
   id 'org.ajoberstar.grgit' version '2.1.0'
   id "com.github.spotbugs" version "2.0.0"
+  id "org.sonarqube" version "2.8"
+}
+
+sonarqube {
```

Comment:
why do we need this ?

Reply:
Surely for nothing, I’ve been playing around with this tool on this branch and I forgot to delete this. Going to remove asap I get back to my computer, thanks for the notice